File: Semester 4/Programming and Projection Enironments/Proiect_MPP/src/main/java/root/proiect_mpp/repositories/people/clients/IClientRepository.java
package root.proiect_mpp.repositories.people.clients;

import root.proiect_mpp.domain.people.Client;
import root.proiect_mpp.repositories.people.IPersonRepository;

public interface IClientRepository extends IPersonRepository<Client> {
    Client findByAddress(String address);
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/Proiect_MPP/repository/people/clients/IClientRepository.cs
using Proiect_MPP.domain.people;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proiect_MPP.repository.people.clients
{
    public interface IClientRepository : IPersonRepository<Client>
    {
        Client? findByAddress(string address);
    }
}
